admin_guide/table_management.rst

```diff
@@ -165,7 +165,6 @@ contain the same data, and compare:
 
 ::
 
-  .
```

Comment:
The reason this dot is here is to set the indentation level of the code block. If the dot isn't present, Sphinx will consider the first line to be the one starting with `   row_size` and will count three spaces of indentation. The horizontal line of dashes is less indented, however, and would end the code block.

The dot thing is a hack, no doubt about that. I just don't know a better way. Maybe you can think of one?

Reply:
Oh sorry, I hadn't realized that! I will put the dot back for now, and will let you know if I find a better way.